lib/mail-archive-helper.ts

```diff
@@ -62,9 +62,8 @@ export class MailArchiveGitHelper {
             return "";
         }
 
-        return "``````````\n" +
-            body + (body.endsWith("\n") ? "" : "\n") +
-            "``````````\n";
+        return "``````````\n"
+            + body + (body.endsWith("\n") ? "" : "\n") + "``````````\n";
```

Comment:
I guess this cannot be converted to a template? ;-)

Reply:
Maybe next time.
```
const separator = "``````````\n";
return `${separator}${body}${body.endsWith("\n") ? "" : "\n"}${separator}`;
```

Reply:
:-)